src/main/java/org/openmrs/addonindex/legacy/LegacyController.java

```diff
@@ -75,13 +75,14 @@ public String checkUpdate(@PathVariable("moduleId") String moduleId) throws Exce
 	@RequestMapping(method = RequestMethod.GET, value = "findModules")
 	public LegacyFindModulesResponse findModules(
 			@RequestParam(value = "sSearch", required = false) String query,
+			@RequestParam(value = "tags", required = false) String tag,
```

Comment:
Replying to your above comment, I actually saw that you didn't explicitly test the search function Inside the Elastic search test file. Moreover you tested the fact as to whether the search feature was working and hence I didn't ​write any tests either